### ISSUE TITLE
Double sameword annotation

### ISSUE THREAD

**Issue by stenskjaer** [stenskjaer/samewords#33]
This is lifted from #29.

```latex
\documentclass{scrartcl}

\usepackage[series={A},nofamiliar,noeledsec,noledgroup,draft]{reledmac}

\begin{document}

\beginnumbering
\pstart
TWO~dollars and
\edtext{TWO \edtext{cent}{%
	\Afootnote{dimes}}
and
even
%
\edtext{TWO}{%
	\Afootnote{4}}
 more}{%
	\lemma{TWO–more}%
	\Afootnote{del.}},
and
%
\edtext{TWO~%
\edtext{dollars}{%
	\Afootnote{cents}}
}{%
	\Afootnote{del.}}
some
more.
\pend
\endnumbering

\end{document}
```

The result gives some double `\sameword{\sameword{` and will not compile with `reledmac`:

```latex
./orig1-SWtestSW.tex:30: Undefined control sequence.
<argument> ... @\this@absline @\the \section@numR 
                                                  @R
l.30 \endnumbering
```

Removing the double `\sameword`s makes it run through with a correct result.

On the other hand 

> 1 TWO⁴ dollars² ] del.

is a bit strange. I'd expect to find

> 1 TWO dollars² ] del.

in an edition.
(But I keep wondering whether one could construct cases where this would be ambiguous. Can you come up with one?)

Considering this, _your_ markup seems to make sense and `reledmac`'s handling of the markup should be different.
What do you think?

**Comment by stenskjaer** [comment]
Okay. This is difficult. I find that the nested samewords is the intuitively correct way of doing this. This has also been discussed (https://github.com/maieul/ledmac/issues/646), and it is implemented so that you should be able to nest the sameword annotations. (§6.3.5) of the handbook I believe.

I therefore also find that the expected disambiguation in an apparatus would be "1 TWO dollars² ] del."

However: It seems that there may be a problem with how Reledmac handles `\edtext{}{}` inside a `\sameword{}`. 

But certainly I don't see any other possible way of annotating this that would convey the situation more correctly.

There may by the way also be a problem with the `~` in side the sameword macro. That itself gives an error. 

So let's try a simplified version:
```
\documentclass{scrartcl}

\usepackage[draft]{reledmac}

\begin{document}

\beginnumbering
\pstart
\sameword{\sameword{two} \sameword{dollars}} and
\edtext{\sameword[1]{two}}{\Afootnote{4}}
\edtext{\sameword[1]{\sameword{two} \edtext{\sameword[2]{dollars}}{\Afootnote{cents}}}}{\Afootnote{del.}}
and more \sameword{dollars}.
\pend
\endnumbering

\end{document}
```

This gives the error 
```
! Missing \endcsname inserted.
<to be read again> 
                   \csname\endcsname 
l.11 ...s}}{\Afootnote{cents}}}}{\Afootnote{del.}}
```

When the `sameword` wrapping the nested edtext is removed it runs. But that is not what we want, as we need the nested apparatus as well as all the sameword annotations. I'm working on an issue for the reledmac repo.

**Comment by stenskjaer** [comment]
This now gives a compilable text, but the critical note is still printed with two numbers. This may not be what the editor wants, but currently it needs to be addressed in reledmac too, if we want to find a good solution to it.

This is a subject of discussion in https://github.com/maieul/ledmac/issues/771.

I close this for now, as the result compiles and the other problems are addressed in other contexts.

**Comment by floriandk** [comment]
I can't find an issue-33 branch. Is this perhaps somehow private…? Anyway I'd be happy to test it.
Or are you about to merge this anyway?

**Comment by stenskjaer** [comment]
No, actually it's not ready for test yet. I wanted to push it tonight (it's just on my local machine now), but there are problems with the wrapping of the sameword annotation, which is related to the problems in #31. So maybe solving one will move the other further along. 
I'll get back to it ASAP.

**Comment by stenskjaer** [comment]
So, this problem was connected with the overlapping and multiword/singleword annotations questions in #31. So now I have pushed an update to "issue-31" that should address this.

When you get the time, could you try to give this a try with the original MWE and see if you also find it to work. It may be relevant to try setting the "multiword" setting in the config-file to confirm that it also works with multiword annotation. Because the single word annotation would not have given the problem in the first case.

Cheers!

**Comment by stenskjaer** [comment]
I have merged the changes into master and released 0.5. I'm closing this for now.
